### ISSUE TITLE
Separate volume of COM1/2

### ISSUE THREAD

**Issue by FlyingFoxVATSIM** [swift-project/pilotclient#100]
_swift_ is not using X-Plane's DataRef to read out the volumes of COM1 and COM2. Would it be possible to implement this for it to have an effect on the output volume?

```
sim/cockpit2/radios/actuators/audio_volume_com1
sim/cockpit2/radios/actuators/audio_volume_com2

```
The volume range is from 0 to 1. In the Zibo B738 the volume steps are 0.05, on other aircraft they are 0.01 or even 0.10

**Comment by oktal3700** [comment]
We actually do read those datarefs in xswiftbus, and we transfer the values to swift. But then swift doesn't use them. It's possible we just forgot.

**Comment by NewoIsTaken** [comment]
Also, would it be possible to do this same thing on FlightGear?

**Comment by jo5ef** [comment]
That'd be a cool feature. I have a RealSimGear G1000 and would use it with

```
sim/GPS/g1000n1_cvol_dn
sim/GPS/g1000n1_cvol_up
```

**Comment by oktal3700** [comment]
Actually xswiftbus does read the datarefs and sends them to swift, but swift doesn't use them yet.

**Comment by tigert** [comment]
For MSFS2020 the volumes are COM VOLUME variable with an index specifying the radio in question:

(A:COM VOLUME:1, Percent)
(A:COM VOLUME:2, Percent)

These are part of the SDK in MSFS2020.

**Comment by tigert** [comment]
**Flightgear** seems to have this in the common property tree:

` /instrumentation/comm[n]/audio/volume-recv `

It has also volume-xmit for radios, but maybe transmission volume should be only adjustable in Swift settings.

**Comment by ltoenning** [comment]
> For MSFS2020 the volumes are COM VOLUME variable with an index specifying the radio in question:
> 
> (A:COM VOLUME:1, Percent) (A:COM VOLUME:2, Percent)
> 
> These are part of the SDK in MSFS2020.

Thanks. I've split this up into #145.

**Comment by ltoenning** [comment]
Separate COM volumes can now be set in the UI and can be controlled from FG and XP.
Available with 0.9.7.149 and 0.9.8.36 (or later).
